forge/db/controllers/Project.js

```diff
@@ -59,8 +61,9 @@ module.exports = {
                 })
             }
         }
-        if (project.ProjectSettings[0]?.key === 'settings') {
-            const projectSettings = project.ProjectSettings[0].value
+        const projectSettingsRow = project.ProjectSettings.find((projectSetting) => projectSetting.key === KEY_SETTINGS)
```

Comment:
I'd really prefer if these finder methods were somehow on the ProjectSettings model, but at runtime project.ProjectSettings` is a naive array object. The closest I could find would be adding a static method to the ProjectSettings prototype, but this breaks Sequelize conventions and feels like a hack.